dlrover/python/scheduler/kubernetes.py

```diff
@@ -351,13 +351,16 @@ def initilize(self):
                 memory = 0
             gpu_type = None
             gpu_num = 0
+            gpu_stats = []
             for k, v in requests.items():
                 if "nvidia.com" in k:
                     gpu_type = k
                     gpu_num = int(v)
             group_resource = NodeGroupResource(
                 num,
-                NodeResource(cpu, memory, gpu_type, gpu_num, priority),
+                NodeResource(
```

Comment:
Why do we need to modify these codes?

Reply:
Now the `NodeResource` is initialized with gpu_stats. We need to add an empty list to initialize the `NodeResource`.

Reply:
But it's also strange here, ideally kubernetes.py should only handle resource configuration requests from yaml. However `gpu_stats` is not part of that.